_data/meltano/extractors/tap-intercom/ticketswap.yml

```
@@ -0,0 +1,78 @@
+capabilities:
+- about
+- batch
+- catalog
+- discover
+- schema-flattening
+- state
+- stream-maps
+description: A Singer Tap for Intercom, built with the Meltano Tap SDK
+domain_url: ticketswap.com
```

```suggestion
domain_url: https://developers.intercom.com/intercom-api-reference/v1.4/reference
```

The other variants also link to this API docs as the domain_url. Does that work?